apiary-metastore-events/apiary-hive-events/src/test/java/com/expediagroup/apiary/extensions/events/metastore/event/ApiaryListenerEventFactoryTest.java

```diff
@@ -129,6 +133,15 @@ public void createSerializableDropPartitionEvent() {
   @Test
   public void createSerializableInsertEvent() {
     InsertEvent event = mockEvent(InsertEvent.class);
+
+    Table table = new Table();
+    table.setPartitionKeys(ImmutableList.of(new FieldSchema("KEY", null, null)));
+    Partition partition = new Partition();
+    partition.setValues(ImmutableList.of("VALUE"));
+
+    when(event.getTableObj()).thenReturn(table);
+    when(event.getPartitionObj()).thenReturn(partition);
```

Comment:
Should we have some assertions on these below since you've gone to the extra work to create them? Alternatively you could just leave out the setting of the partition keys here and this test would effectively be testing the "no keys" case (which I guess is what it was doing before?)